components/button/button.css

```diff
@@ -0,0 +1,125 @@
+@import url("../../css/navbar.css");
+@import url("../../css/sidebar.css");
+
+/***** MAIN CONTENT*/
+
+.example-box {
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+.btn {
+    padding: 0.5rem 1rem;
+    border-radius: 5px;
+    border: none;
+    color: var(--font-white);
+    font-family: inherit;
+    text-decoration: none;
+    font-size: inherit;
+}
+
+.btn-icon {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+}
+
+.primary {
+    background-color: #3454D1;
+}
+
+.secondary {
+    background-color: #71717a;
+}
+
+.success {
+    background-color: #32cd32;
+}
+
+.danger {
+    background-color: #dc2626;
+}
+.warning {
+    background-color: #f59e0b;
+}
+.dark {
+    background-color: var(--secondary-color);
+}
+
+.primary-ol {
+    border: 1px solid #3454D1;
+    color: #3454D1;
+    background: #ffffff;
```

Comment:
background: inherit 
will be better for outlined btns as it will work with every background